program/src/cmd/info.rs

```diff
@@ -107,7 +107,7 @@ impl fmt::Debug for Mail {
             )
             .field(
                 "payload",
-                &["0x", &hex::encode(&self.message.payload)].concat(),
+                &["0x", &hex::encode(&self.message.payload.0)].concat(),
```

Comment:
Maybe it is worth implementing `Deref`, `AsRef`, and similar traits for `LimitedVec` to avoid such changes in the future.

Reply:
As you can see in changes, most of code places use `Vec<u8>`, not the `&[u8]`. That's the only place, where trait is required.

Reply:
We could say that changes in `program/tests/*` can be simplified with these traits, but it requires changing the `messager::REPLY_REPLY.encode()` to `&messager`, which shows that implementing suggested traits doesn't make significant changes.
We can consider doing that in the future with bringing changes to types used in `gear-program`